teachrecovery/templates/base.html

```diff
@@ -34,6 +34,25 @@
    };
   </script>
 
+<script src="//cdn.ravenjs.com/1.1.18/jquery,native/raven.min.js"></script>
```

Comment:
I'm finding that setting this up for non-debug only is a good thing, e.g. {% if not debug %}